app/formatters.py

```diff
@@ -46,39 +48,50 @@ def format_time(date):
 
 
 def format_datetime_normal(date):
-    return "{} at {} UTC".format(format_date_normal(date), format_time_24h(date))
+    return "{} at {} {}".format(
+        format_date_normal(date), format_time_24h(date), get_user_preferred_timezone()
+    )
 
 
 def format_datetime_short(date):
-    return "{} at {} UTC".format(format_date_short(date), format_time_24h(date))
+    return "{} at {} {}".format(
+        format_date_short(date), format_time_24h(date), get_user_preferred_timezone()
+    )
 
 
 def format_datetime_relative(date):
-    return "{} at {} UTC".format(get_human_day(date), format_time_24h(date))
+    return "{} at {} {}".format(
+        get_human_day(date), format_time_24h(date), get_user_preferred_timezone()
+    )
 
 
 def format_datetime_numeric(date):
-    return "{} {} UTC".format(
-        format_date_numeric(date),
-        format_time_24h(date),
+    return "{} {} {}".format(
+        format_date_numeric(date), format_time_24h(date), get_user_preferred_timezone()
     )
 
 
 def format_date_numeric(date):
     date = parse_naive_dt(date)
-    return date.strftime("%Y-%m-%d")
+
+    et = pytz.timezone(get_user_preferred_timezone())
```

Comment:
I realize we are going for Eastern Time initially with this, but with the intent to change to the user's preference afterward.

Let's change references of `et` to `user_timezone` so that it's clear what this is and will be in the future.